app/views/lists/map_view.html.erb

```diff
@@ -37,16 +37,42 @@ var voters = <%= @geojson.html_safe %>
 
 var votersLayer = L.mapbox.featureLayer(voters).addTo(map);
 
+var lists = [];
+
 map.fitBounds(votersLayer.getBounds());
 
-map.on('draw:created', showPolygonArea);
+map.on('draw:created', findVoters);
 
-function showPolygonArea(e) {
+function findVoters(e) {
   var new_layer = map.addLayer(e.layer);
-  if (leafletPip.pointInLayer(marker.geometry.coordinates, new_layer).length > 0) {
-    alert("The point is in one of your shapes");
-  } else {
-    alert("The point is outside your shapes");
+  array = []
+  for (var i = 0; i < voters.length; i++) {
+    if (leafletPip.pointInLayer(voters[i].geometry.coordinates, new_layer, true)[0] === e.layer) {
+      array.push(voters[i]);
+    };
   };
-}
+  lists.push(array)
+};
+
+$(document).ready(function(){
+  $(".save").click(function(e){
+    e.preventDefault();
+    var listsName = prompt("Enter a name for these lists");
+    var listsData = []
+    for (var i = 0; i < lists.length; i++) {
+      var list = lists[i];
+      var listName = listsName + " Turf " + (i + 1);
+      var voterIDs =[];
+      for (var j = 0; j < list.length; j++) {
+        voter = list[j];
+        voterIDs.push(voter.properties.voter_id);
+      };
+      listsData.push({ name: listName, voter_ids: voterIDs });
+    }
+    $.ajax({type: "POST",
+      url: "/map_lists_create",
+      data: { lists: listsData }
+    })
```

Comment:
you should have success and error callbacks on this AJAX routine to provide the user with feedback that the save successful or not successful.